src/js/features/ingestion/ingestion.store.ts

```diff
@@ -15,18 +15,18 @@ export const makeGetIngestionDataRequest = createAsyncThunk<LastIngestionRespons
 
 export interface IngestionDataState {
   isFetchingIngestionData: boolean;
-  IngestionData: IngestionData[];
+  ingestionData: ingestionData[];
```

Comment:
type still should have a capital 'I'!

this should be `ingestionData: IngestionData[];`